portable/GCC/ARM_CM7/r0p1/port.c

```diff
@@ -525,14 +525,20 @@ void xPortSysTickHandler( void )
      * save and then restore the interrupt mask value as its value is already
      * known. */
     portDISABLE_INTERRUPTS();
+    traceISR_ENTER();
```

Comment:
The macro name needs to be more specific if it is only used for the SysTick interrupt.  There is no central interrupt handler on ArmV7-M and ArmV8-M so perhaps giving the macro a SysTick specific name is a good idea anyway?

Reply:
Good point. I will rename it to traceISR_SYSTICK_ENTER and traceISR_SYSTICK_EXIT.